UsingExoPlayer/app/build.gradle

```diff
@@ -0,0 +1,38 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion compileSdkVer
+    buildToolsVersion buildToolsVer
+
+    defaultConfig {
+        applicationId "github.nisrulz.example.exoplayer"
+        minSdkVersion minSdkVer
+        targetSdkVersion targetSdkVer
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility 1.8
+        targetCompatibility 1.8
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    // Testing-only dependencies
+    testImplementation "junit:junit:$junitVer"
+    androidTestImplementation "com.android.support.test:runner:$androidTestRunnerVer"
+    androidTestImplementation "com.android.support.test.espresso:espresso-core:$espressoCoreVer"
+
+    implementation "com.android.support:appcompat-v7:$supportLibVer"
+    implementation 'com.google.android.exoplayer:exoplayer-core:2.9.0'
```

Comment:
the version of  exo player `2.9.0` should be put in `dependencies.gradle` file and then referenced here.

Reply:
Got it, I'll add the line where it belongs. I used Java because these are examples I had laying around so I quickly adapted/updated them for the repository guidelines. I can try to convert them to Kotlin (I don't think there will be any issue) but it will take me a bit.